### DISCUSSION TITLE
PoMenuItem: Não renderizar, dado uma condição.

### DISCUSSION THREAD

**Discussion by CarlosEReis** [po-ui/po-angular#1889] [Tire suas dúvidas] [answered]
Olá!
Gostaria de saber se existe alguma propriedade no **PoMenuItem** no qual dado uma condição boleana, o item seja renderizado ou não.

**Exemplo**
Eu tenho um único menu, e dois tipos de usuários (admin e user). Apenas o usuário com a permissão admin pode visualizar o item "Usuários". 

``` javascript
readonly menu: Array<PoMenuItem> = [
    { label: 'Home', link: 'dashboard', icon: 'po-icon po-icon-chart-area', shortLabel: 'Home' },
    { label: 'Clientes', link: 'clientes', icon: 'po-icon po-icon-handshake', shortLabel: 'Clientes' },
    { label: 'Chamados', link: 'chamados', icon: 'po-icon po-icon-news', shortLabel: 'Chamados' },
    { label: 'Usuários', link: 'usuarios', icon: 'po-icon po-icon-users', shortLabel: 'Usuários' }
  ];

```

**Comment by anliben** [comment, marked as answer]
Boa Tarde @CarlosEReis, tudo bem?

Antes de enviar esses itens para o menu podes fazer uma validação pelo tipo do cliente ou usuário, atualmente não temos uma validação relacionada a esta dúvida!

Attr.

**Reply by CarlosEReis** [reply]
Fala @anliben! Tudo bem e você?

Ah entendi, tudo bem, vou implementar aqui da forma que sugeriu. 

Obrigado!
